doc/howto/instances_routed_nic_vm.md

```diff
@@ -23,7 +23,13 @@ To configure the gateways with `cloud-init`, firstly initialize an instance:
       "type": "virtual-machine"
     }'
 ```
+````{group-tab} UI
+```{figure} /images/UI/routed_nic_create_instance.png
```

Comment:
This doesnt seem routed nic specific, can we re-use this image across different device types?

Reply:
I'm trying to keep the number of screenshots as low as possible.
The reason I added one here was simply the machine name, to avoid confusion.

But I'll make sure to not use more than needed. :)